programs/server/config.xml

```diff
@@ -21,7 +21,7 @@
 
             [1]: https://github.com/pocoproject/poco/blob/poco-1.9.4-release/Foundation/include/Poco/Logger.h#L105-L114
         -->
-        <level>trace</level>
+        <level>warning</level>
```

Comment:
This is wrong.

Reply:
Ah, just an error, will rebase shortly (before transitioning PR to ready)